microsoft-edge/webview2/Roadmap.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Learn what is coming next for WebView2
+title: Roadmap for Microsoft Edge WebView 2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/19/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Microsoft Edge WebView2 Roadmap
+
+The WebView2 control that allows developers to embed web technologies in their native applications. This document outlines the prospective roadmap for WebView2 through 2020 and into 2021. WebView2 is under active development and the roadmap will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here aren't exhaustive and are subject to change. 
```

Comment:
Call out here that our roadmap is a living document and add the date which the document is last updated. The page is newly constructed, and we'll fill this with more details going forward.